### ISSUE TITLE
Importer: Giving an absolute path as URL should prevent adding baseFile

### ISSUE THREAD

**Issue by kerimoyle** [cellml/libcellml#695]
If we pass a baseFile into the importer when resolving a model which has imports specified with an absolute path, the baseFile still gets added to the path, and makes it nonsense.  

The `Importer` should check and distinguish between relative and absolute URLs, and not add the baseFile path to any absolute ones.

**Comment by nickerso** [comment]
likely duplicate of #592 (or at least the same underlying issue)

**Comment by kerimoyle** [comment]
Yep, same underlying issue.  I knew there was another one out there, thanks :)
